doc/neo-tree.txt

```diff
@@ -148,6 +148,38 @@ commands by adding this option before the Neotree config is run:
 >vim
   let g:neo_tree_remove_legacy_commands = 1
 <
+CALLING_FROM_LUA
+
+You can call Neotree commands from your Lua scripts as follows:
+
+>lua
+  require('neo-tree.command').execute({ ... })
+
+For example, in the `keys` configuration, you could use the following to make
+the `-` key reveal the current file, or if in an unsaved file, the current
+working directory.
+
+>lua
+
+  keys = {
+    {'-', function()
+      local reveal_file = vim.fn.expand('%:p')
+      if (reveal_file == '') then
+        reveal_file = vim.fn.getcwd();
+      else
+        local f = io.open(reveal_file, "r")
+        if (f) then
+          f.close(f)
+        else
+          reveal_file = vim.fn.getcwd()
+        end
+      end
+      require('neo-tree.command').execute({
+        reveal_file = reveal_file, reveal_force_cwd = true
+      })
```

Comment:
If we are providing an example as documentation I would change two things here:

1. Put each option on its own line.
2. Provide each optional argument as well as an example.

```suggestion
      require('neo-tree.command').execute({
        action = "focus", -- OPTIONAL, this is the default value
        source = "filesystem", -- OPTIONAL, this is the default value
        position = "left", -- OPTIONAL, this is the default value
        reveal_file = reveal_file, -- path to file or folder to reveal
        reveal_force_cwd = true, -- change cwd without asking if needed
      })
```